docker/test

```diff
@@ -15,41 +17,45 @@ OPTIONS:
   -o OS     Specify operating system where candlepin will be running
               For postgres are supported:
                 cs8 - Centos Stream 8 (default)
+                cs9 - Centos Stream 9
               For mysql are supported:
                 cs8 - Centos Stream 8 (default)
+                cs9 - Centos Stream 9
               Without database are supported:
                 cs8 - Centos Stream 8 (default)
+                cs9 - Centos Stream 9
               For sonarqube are supported:
                 cs8 - Centos Stream 8 (default)
+                cs9 - Centos Stream 9
   -m        use mysql
-  -n NAME   Sets the project name for the docker-compose run
+  -n NAME   Sets the name for the pod where containers will be running
+              and also is used as prefix for containers
   -p        use postgres
   -x        use no database. only spin up the candlepin container
               (useful for running things like unit tests, checkstyle,
               that don't need a running server with db)
   -q        use no database, but require the sonarqube certificate
               (for running a sonarqube upload)
-  -l        skip docker pull and use local images
+  -l        skip podman pull and use local images
   -d        run containers detached and do not
               automatically shut down & remove the
               containers on the script's exit
-  -e        expose ports 8443, 8080 and 22 to the host machine
+  -e        expose ports 8443 and 8080 to the host machine
 
 USAGE
 }
 
 while getopts ":c:mn:o:pxqlde" opt; do
   case $opt in
     c) TEST_CMD="$OPTARG";;
-    m)
-       COMPOSE_ARGS="-f $DIR/docker-compose-mysql";
+    m) DB="mariadb:10.2";
        chcon -Rt svirt_sandbox_file_t $DIR/mysql.cnf;
        sudo chown 999:999 $DIR/mysql.cnf;;
-    n) PROJ_NAME="-p $OPTARG";;
+    n) PROJ_NAME=$OPTARG;;
     o) OPERATING_SYSTEM=$OPTARG;;
-    p) COMPOSE_ARGS="-f $DIR/docker-compose-postgres";;
-    x) COMPOSE_ARGS="-f $DIR/docker-compose-no-db";;
-    q) COMPOSE_ARGS="-f $DIR/docker-compose-sonarqube";;
+    p) DB="postgres:12";;
+    x) DB="";;
+    q) SONARQUBE="1";;
```

Comment:
We don't need options x and q in master branch any more (removed by Ondrej's PR)

Reply:
(but cp-test needs then for old branches to still function)

Reply:
i see i am just rebasing it

Reply:
done